src/ORM/FieldType/DBMoney.php

```diff
@@ -39,7 +39,7 @@ public function getFormatter(): NumberFormatter
     public function Nice(): string
     {
         if (!$this->exists()) {
-            return null;
+            return '';
```

Comment:
This is unrelated, though clearly wrong. All other DBField subclasses that implement Nice() have a `string` return type so I think this is correct